src/elm/Main.elm

```diff
@@ -92,6 +92,7 @@ type Msg
     | HideToast Int
     | StorageDocReceived D.Value
     | SetCookieConsent Bool
+    | Never
```

Comment:
I think some documentation on why this variant is needed would be good.

Reply:
I have renamed never to Nop, since this is a more precise name
I have also added a comment describing the purpose